fmlloader/src/main/java/net/minecraftforge/fml/loading/VersionSupportMatrix.java

```diff
@@ -10,25 +10,25 @@
 import org.apache.maven.artifact.versioning.VersionRange;
 import org.jetbrains.annotations.ApiStatus;
 
+import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 import java.util.function.BiPredicate;
 
 @ApiStatus.Internal // since 1.21.1, will be made non-public in a later MC version
 public class VersionSupportMatrix {
-    private static final boolean ENABLED = FMLLoader.versionInfo().mcVersion().equals("1.21.1");
-    private static final Map<String, List<ArtifactVersion>> overrideVersions;
+    private static final Map<String, List<ArtifactVersion>> OVERRIDE_VERSIONS;
 
     static {
-        if (ENABLED) {
-            overrideVersions = Map.ofEntries(
+        if (FMLLoader.versionInfo().mcVersion().equals("1.21.1")) {
```

Comment:
Only other slight tweak is I prefer to do "String".equals(variable) as that prevents any possibility of a NPE. Not an issue just a style note.